src/utils/utils.ts

```diff
@@ -321,3 +322,11 @@ export const batchDetailsOptionalDisplay = (data: string) => {
       return detailsPageOptionalDisplay(data);
   }
 };
+
+export const toastifyCustomStyle: ToastOptions<{}> = {
+  hideProgressBar: true,
+  autoClose: false,
+  theme: 'dark',
+  position: toast.POSITION.BOTTOM_CENTER,
+  toastId: 'custom-toast'
```

Comment:
intentional in using the same toast id?  AFAIK this causes the toasts to override each other.

Reply:
resolved